data_processing/fixed_cost_assumptions.md

```diff
@@ -0,0 +1,55 @@
+## Scenario 11: Addition of fixed costs
+by Olek Yardas
+
+Fixed costs for non-automobile technologies came from Report SL-014940, a capital cost study by Sargent & Lundy commissioned by USEIA. Fixed costs for automobile technologies comes from averaging MSRP data for cars from fueleconomy.gov.
+
+### General Assumptions for fixed costs for each technology
+
+#### Capital costs
+Capital costs for a given technology are divided into owner's costs, and engineering, procurenent, and construction (EPC) contractor costs. Capitcal costs account for all costs incurred during construction of the power plant before coming online.
+The S&L report makes the following assumptions for the capital costs:
+ - The power plant developer/owner hires an EPC contractor for construction.
+ - EPC contractor cost includes procurement of equipment, materials, and all construction labor.
+ - The owner costs includes project development, permitting, legal, outside-the-fence-line costs (electrical/gas interconnection), contingency, etc
+ - Electrical interconnecction does not require substation upgrades.
+
+Locational and environmental adjustment factors are in the appendices of the paper.
+
+#### Operating and maintenance costs.
+Fixec O&M costs include routine labor, materials and contract services, and administrative and general expenses. For solar and wind all O&M costs are treated as fixed costs.
+
+#### Electrolysis plant (ELECTROL)
+The capital investment cost for electrolysis cells comes from the 2018 paper "The investment cost of electrolysis - A comparison of cost studies from the past 30 years". In this paper, the estimated investment cost of alkaline electrolyzers for the future (2030) is between 787 and 906 euros/kW output. According to the "Technology Roadmap Hydrogen and Fuel Cells", the efficienty of alkaline electrolyser cells is between 65 and 82%. The average of these costs is 846.5 euros/kW output. In USD this is $1030.19 $/kW
+
+The fixed cost is much trickier to determine, and I have not found a good estimate for this yet.
+#### Nuclear power plant (NUCLEAR)
+
+This case uses two Westinghouse AP1000 reactors as a model plant.
+Fixed O&M costs come from those listed at the top of this document.
+
+#### Wind power (IMPWIND)
+
+This case is a 200MW capacity onshore wind farm located in the Great Plains region. Land is abundant and suitable for turbine siting.
```

Comment:
We have a wind farm. Railsplitter in Lincoln, IL

Reply:
The wind farm from the S&L report is comparable in size and capacity. Should I try to find a second source for cost estimates for the wind farm?

Reply:
Question about the wind farm: we purchase like 8% of the power they generate, which is a variable cost, no? So if we are modelling from the perspective of the university we only have that variable cost really.

Reply:
^^ @samgdotson

Reply:
We need to account for the PPA cost bc UIUC is not an operator/maintainter of Rail Splitter